.gitignore

```diff
@@ -1,2 +1,7 @@
 /dist/
 /dist-newstyle/
+/.cabal-sandbox/
+TAGS
```

Comment:
With a global `.gitignore` you might end up ignoring more stuff than strictly needed as your global `.gitignore` becomes the union of `.gitignore`s for all projects you're working on. Per-project ones also provide a better experience for new contributors.